Linux Script/configure-linux.sh

```diff
@@ -924,33 +927,82 @@ getPassword()
 	echo
 }
 
-#Change TLS settings
-checkIfTLS()
+#function to switch system logging to insecure mode if user runs the modular script in insecure mode
+switchSystemLoggingToInsecure()
 {
-   if [[ $LOGGLY_SYSLOG_PORT == 514 ]]; then
-   
-        if [ "$SUPPRESS_PROMPT" == "false" ]; then
-	        while true;
-			do
-	            read -p "Hey you are going to setup system logs in insecure mode. Do you want to overwrite this with secure mode? (yes/no)" yn
-						case $yn in
-						[Yy]* )
-							logMsgToConfigSysLog "INFO" "INFO: Going to overwrite the conf file: $LOGGLY_RSYSLOG_CONFFILE with secure configuration";
-							LOGGLY_TLS_SENDING="true"
-							LOGGLY_SYSLOG_PORT=6514
-							break;;
-						[Nn]* )
-							break;;
-						* ) echo "Please answer yes or no.";;
-						esac
-			done			
-	    else
-		    logMsgToConfigSysLog "WARN" "WARN: Your system logs are being send insecurely. We prefer to send system logs securely so switching to secure configuration."
-			LOGGLY_TLS_SENDING="true"
-			LOGGLY_SYSLOG_PORT=6514
-			
-	    fi
-    fi		
+	if [ -f $LOGGLY_RSYSLOG_CONFFILE ]; then
+		if [[ $RSYSLOG_VERSION_TMP -le "7" ]]; then
+				EXISTING_SYSLOG_PORT=$(grep 6514 $LOGGLY_RSYSLOG_CONFFILE | awk {'print $2'} | cut -d':' -f2 | cut -d';' -f1)
+		elif [[ "$RSYSLOG_VERSION_TMP" -ge "8" ]]; then
```

Comment:
I'm not sure you should switch this off the version number, since the newer versions are also compatible with the legacy syntax

Reply:
@mostlyjason Actually the TLS configuration for rsyslog version **less than 7** and **greater than 8** is different and since I need to grep the port 6514 from the TLS configuration of 22-loggly.conf file, I had to use two grep commands to grep port 6514 from different places in both the configurations. 

TLS configuration for rsyslog version less than 7 can be see here- https://github.com/loggly/install-script/blob/master/Linux%20Script/configure-linux.sh#L521-L546

TLS configuration for rsyslog version greater than 8 can be see here- https://github.com/loggly/install-script/blob/master/Linux%20Script/configure-linux.sh#L549-L569

That is why I used two different commands to grep the 6514 port based on rsyslog versions. 

Also, the NON-TLS configuration is same for any rsyslog version so I used only one grep command to pick the port 514 from **22-loggly.conf** file.